test/micro/org/openjdk/bench/java/io/ObjectInputStreamTest.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2020, 2024, Red Hat Inc. All rights reserved.
```

Comment:
Probably the result of a copy&paste...
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```